lib/help_scout.rb

```diff
@@ -1,6 +1,8 @@
 require "help_scout/version"
 require "httparty"
 
+class HelpScout::ValidationError < StandardError; end
```

Comment:
Ah thought that was only possible inside a `Module`, but seems to be working :+1: